src/main/java/com/nike/cerberus/operation/composite/RotateCertificatesOperation.java

```diff
@@ -50,31 +65,54 @@ public RotateCertificatesOperation(CloudFormationService cloudFormationService)
         commandList.addAll(Lists.newArrayList(
                 // Add the cert and key files to S3
                 ChainableCommand.Builder.create().withCommand(new UploadCertificateFilesCommand()).build(),
-                // Add the new cert to the ALB
-                ChainableCommand.Builder.create().withCommand(new AddCertificateToAlbCommand()).build(),
+                // Update the Load Balancer stack to use the new cert
+                ChainableCommand.Builder.create().withCommand(new UpdateStackCommand())
+                        .withAdditionalArg(UpdateStackCommand.STACK_NAME_LONG_ARG)
+                        .withAdditionalArg(Stack.LOAD_BALANCER.getName())
+                        .build(),
                 // Generate new CMS config that points to the new cert
-                ChainableCommand.Builder.create().withCommand(new CreateCmsConfigCommand()).build(),
+                ChainableCommand.Builder.create().withCommand(new UpdateCmsConfigCommand()).build(),
                 // Do a rolling reboot of the management service
-                ChainableCommand.Builder.create().withCommand(new RollingRebootWithHealthCheckCommand()).build()
-                // Delete the old cert from the ALB and from the Identity Management service and S3
-
-                // Revoke the Cert? todo
+                ChainableCommand.Builder.create().withCommand(new RebootCmsCommand()).build()
         ));
-
         return commandList;
     }
 
+    @Override
+    protected void beforeChain() {
+        log.info("Preparing to rotate certificates");
+    }
+
+    @Override
+    protected void afterChain() {
```

Comment:
Why afterChain() rather just another step in the chain?

Reply:
It is nice for doing some simple logging so that I dont have to create a command and operation and help message and register it with the CLI runner, just to do a couple simple things.

Reply:
The logging maybe makes sense.  

The delete here would be better as a command:
- So it can be ran independently.  
- Also, for error handling, you could say the command that failed, it would be more obvious without going to the source code as a user.
- Gives composite commands a simpler design.

Reply:
I can create a command just for this, but probably not register it as a single runnable command and add it to the chain.

- [ ] Create command for cert clean up